### PR TITLE
Feature/teamAPI

### PR DESCRIPTION
**Pull request by Blackross22**
Added teamAPI(CRUD)

### CODE REVIEW

File: api.js [hunk 1]
```diff
@@ -18,9 +19,10 @@ api.use(express.json())
 api.use("/api/v1/", todoAPI)
 api.use("/api/v1/", checklistAPI)
 api.use("/api/v1/", userAPI)
+api.use("/api/v1/", teamAPI)
 
 //Index
-api.get("/", (req, res)=>{
+api.get("/", async (req, res)=>{
     console.log("INDEX")
```
Comment (Icyscools) [inline api.js:26]: Remove console.log

**Review by Icyscools — CHANGES_REQUESTED** [review]

File: api.js (hunk 1 above)
Comment (Icyscools) [reply api.js:26]: And why it be async function?

Comment (trabnm1313) [reply api.js:26]: once use it as a command testing function and forget to delete 'async'​ out of it, we will fix it next commit. :)​

Comment (Icyscools) [reply api.js:26]: Ok. You can push to `feature/teamAPI` branch, don't need to create new PR na.

**Pull request merged**